plugins/schedule/base/lib/api/KalturaScheduleEvent.php

```diff
@@ -307,6 +342,25 @@ public function validateForInsert($propertiesToSkip = array())
 		parent::validateForInsert($propertiesToSkip);
 	}
 	
+	protected function setTimingFromLinkedToEvent($linkedToEventId, $duration = null)

```

Comment:
Duration can be sent in all the use case, this function must be deterministic, having duration at all time (remove the null)